src_c/key.c

```diff
@@ -709,6 +709,37 @@ key_name(PyObject *self, PyObject *args)
 #endif
 }
 
+static PyObject *
+key_code(PyObject *self, PyObject *args, PyObject *kwargs)
+{
+    const char * name;
+
+    static char *kwids[] = {
+        "name",
+        NULL
+    };
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwargs, "s", kwids, &name))
+        return NULL;
+
+#if IS_SDLv1
+    PyErr_SetString(PyExc_NotImplementedError, "not supported with SDL 1");
+    return 0;
+#else
+    long code = SDL_GetKeyFromName(name);
```

Comment:
Can you try to add `long code;` under `const char * name;` and then replace this line with `code = SDL_GetKeyFromName(name);`. For some reasons sometimes on python 2 you are not allowed to declare variables after the start of function and that could be the reason why AppVeyor is failing with this weird syntax errors

Reply:
I did this too.